Makefile

```diff
@@ -1,6 +1,6 @@
 PROJECTS = $(wildcard ./client/* ./server/*)
-OPA_IMAGE ?= docker.io/openpolicyagent/opa:latest
-OPA ?= docker run -v ${PWD}/policies:/w/policies -w /w ${OPA_IMAGE}
+OPA_IMAGE ?= ghcr.io/styrainc/enterprise-opa:latest
+OPA ?= docker run -v ${PWD}/policies:/w/policies:rw -w /w ${OPA_IMAGE}
```

Comment:
These are remnants of fixing [this](https://github.com/StyraInc/styra-demo-tickethub/pull/593/files#r1989200744). Not too bad to keep, though, I think.